Source/Atomic/IO/PackageFile.h

```diff
@@ -86,7 +86,20 @@ class ATOMIC_API PackageFile : public Object
     const Vector<String> GetEntryNames() const { return entries_.Keys(); }
 
     // ATOMIC BEGIN
-    
+
+    /// Return a file name in the package, for scripting use ( GetEntryNames is not bindable? )
+    const String& GetEntryName(unsigned index) const 
+    {
+        unsigned nn = 0;  // .Begin() + index syntax not supported on HashMap
+        for (HashMap<String, PackageEntry>::ConstIterator j = entries_.Begin(); j != entries_.End(); ++j)

```

Comment:
```Vector<String>``` won't be picked up currently by bindings (and in general marshaling by an index accessor method can have some benefits over copying arrays/Vectors in terms of speed and dealing with outdated data).    

How about "Return a file name in the package at the specified index" as comments do show up in documentation?

I had a bit of a think on this too, as I am not sure we can count on the HashMap iterator being ordered, though if the package file is being modified while you are iterating, you probably have bigger problems :)